tests/measurements/test_sample.py

```diff
@@ -500,6 +499,37 @@ def circuit(x):
     )
 
 
+@pytest.mark.jax
+@pytest.mark.parametrize(
+    "obs",
+    [
+        # Single observables
+        (qml.PauliX(0)),
+        (qml.PauliY(0)),
+        (qml.PauliZ(0)),
+        (qml.Hadamard(0)),
+        (qml.Identity(0)),
+    ],
+)
+def test_jitting_with_sampling_on_different_observables(obs):
+    """Test that jitting works when sampling observables (using their eigvals) rather than returning raw samples"""
+    import jax
+
+    jax.config.update("jax_enable_x64", True)
+
```

Comment:
Does this statement work here? I was [under the impression](https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#double-64bit-precision) jax fails to use this if not done top-level and before other imports?

Reply:
It does look that way on that documentation page. We use it in a lot of tests though... is this the thing that changed recently? @albi3ro 

Tbh I copy-pasted it without questioning, this is a modified version of the test just above it. Should I remove it and see what happens?

Reply:
We can likely leave as is for now, revisit when multi-precision tests need updating.